### DISCUSSION TITLE
sui package 地址

### DISCUSSION THREAD

**Discussion by v1xingyue** [NonceGeek/Move-Camp#268] [Q&A] [answered]
<img width="352" alt="image" src="https://user-images.githubusercontent.com/974169/216771913-2472fd72-dced-48c6-b66f-98597942b486.png">

这个package 地址 必须是  0x0 么 还是说 是 当前账户的地址?

**Comment by 99Kies** [comment, marked as answer]
因为sui是把合约全部归集到0x0地址下，然后每个合约都是一个object对象（都有对应的objectid）。

sui和aptos的相同点
data和code都是分离的，

sui和aptos在这有个不同点
Aptos用户合约package地址是合约部署者的地址, 而Sui则将所有用户合约地址统一到0x0地址下.

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/45918704/216772187-2dfc57d3-d2c7-4778-b0f3-8c9395666e3c.png">
